src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -391,9 +391,9 @@ examples below to get you started.
 #### Using ChromeDriver
 
 [ChromeDriver](https://sites.google.com/a/chromium.org/chromedriver/) 2.3.0
```

Comment:
nit: we refer to versions in the format 2.30 and not 2.3.0

Reply:
done